status/missing_pods.go

```diff
@@ -22,28 +22,60 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+// DefaultClientPool is a key in the NodeCountByPool map on the LoadTestMissing
+// struct. It maps to the number of nodes required from the default client pool.
+const DefaultClientPool = "default-client-pool"
+
+// DefaultDriverPool is a key in the NodeCountByPool map on the LoadTestMissing
+// struct. It maps to the number of nodes required from the default driver pool.
+const DefaultDriverPool = "default-driver-pool"
+
+// DefaultServerPool is a key in the NodeCountByPool map on the LoadTestMissing
+// struct. It maps to the number of nodes required from the default server pool.
+const DefaultServerPool = "default-server-pool"
+
 // LoadTestMissing defines missing pods of LoadTest.
 type LoadTestMissing struct {
 	// Driver is the component that orchestrates the test. If Driver is not set
 	// that means we already have the Driver running.
-	Driver *grpcv1.Driver `json:"driver,omitempty"`
+	Driver *grpcv1.Driver
 
 	// Servers are a list of components that receive traffic from. The list
 	// indicates the Servers still in need.
-	Servers []grpcv1.Server `json:"servers,omitempty"`
+	Servers []grpcv1.Server
 
 	// Clients are a list of components that send traffic to servers. The list
 	// indicates the Clients still in need.
-	Clients []grpcv1.Client `json:"clients,omitempty"`
+	Clients []grpcv1.Client
+
+	// NodeCountPyPool is a map which gives the number of nodes required from
```

Comment:
NodeCountPyPool --> NodeCountByPool

Reply:
NodeCountByPool means numbers we still missing or what we need in total? Seems like it is the number that we still missing, I figured this out reading codes below, I think it might be better to update the comment to make it clearer. :)